quiche/include/quiche.h

```diff
@@ -352,6 +352,19 @@ ssize_t quiche_conn_send(quiche_conn *conn, uint8_t *out, size_t out_len,
 // Returns the size of the send quantum, in bytes.
 size_t quiche_conn_send_quantum(const quiche_conn *conn);
 
+// Writes a single QUIC packet to be sent to the peer from the specified
+// local address from to the destination address to.
+ssize_t quiche_conn_send_on_path(quiche_conn *conn, uint8_t *out, size_t out_len,
```

Comment:
The rust function's params are `from` and `to`, but here they are `local_addr` and `peer_addr`. Minimally, that means that the comment does't really much sense - to start with I thought this was a type due to lack of decoration around the parameters.

I think for consistency, keeping the from/too notions are going to help. WDYT?

Reply:
@LPardue true... addressed.